kuzzleio/kuzzle
# Seo upgrade

## How should this be manually tested?


  - Step 1 : merge and build with actions
  - Step 2 : check titles and meta on the official doc core
